RetroBar/Taskbar.xaml.cs

```diff
@@ -58,20 +67,25 @@ public Taskbar(WindowManager windowManager, ShellManager shellManager, StartMenu
             DataContext = _shellManager;
             StartButton.StartMenuMonitor = startMenuMonitor;
 
-            DesiredHeight = Settings.Instance.TaskbarScale * (Application.Current.FindResource("TaskbarHeight") as double? ?? 0);
+            
+            _unlockedMargin = Settings.Instance.TaskbarScale * (Application.Current.FindResource("TaskbarUnlockedSize") as double? ?? 0);
+            DesiredRowHeight = Settings.Instance.TaskbarScale * (Application.Current.FindResource("TaskbarHeight") as double? ?? 0);
             DesiredWidth = Settings.Instance.TaskbarScale * (Application.Current.FindResource("TaskbarWidth") as double? ?? 0);
+            DesiredHeight = DesiredRowHeight * Rows;
+            MinHeight = DesiredRowHeight;
```

Comment:
MinHeight doesn't seem to be needed, so I think it should be removed. Also, since it isn't being updated in `RecalculateSize()`, things break :)

Reply:
Sorry. Another hold over from drag resizing